region/norfair/west/Ice Beam Gate Room.json

```
@@ -1004,6 +1004,29 @@
         }
       }
     },
+    {
+      "link": [4, 4],
+      "name": "Leave Shinecharged",
+      "requires": [
+        "Morph",
+        {"canShineCharge": {
+          "usedTiles": 26,
+          "openEnd": 1
+        }},
+        "canShinechargeMovementTricky"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 45
```

```suggestion
          "framesRemaining": 50
```
My best was 57 but it depends so much on your slide into the morph tunnel.